### PR TITLE
Add Songbird Chain-19 support

### PR DESCRIPTION
**Pull request by Uttam-Singhh**

### CODE REVIEW

**Comment by Uttam-Singhh** [comment]
Hey @marcocastignoli 
What's wrong here?

**Comment by marcocastignoli** [comment]
Probably just a connection error

**Comment by marcocastignoli** [comment]
Ok now it passed :)

File: test/chains/chain-tests.js [hunk 1]
```diff
@@ -1230,14 +1230,32 @@ describe("Test Supported Chains", function () {
...
+  
   // Flare Mainnet
   verifyContract(
     "0xbBc2EdeDc9d2d97970eE20d0Dc7216216a27e635",
     "14",
     "Flare Mainnet",
-    ["shared/1_Storage.sol"],
-    "shared/1_Storage.metadata.json"
-  );
+    ["shared/WithImmutables.sol"],
```
Comment (marcocastignoli) [inline test/chains/chain-tests.js:1256]: here you are checking again withImmutables, is that right?

**Pull request merged**